pkg/adapter/adapter.go

```diff
@@ -302,56 +305,127 @@ func (a *Adapter) PollForMessages(channel *wabbit.Channel,
 	}
 }
 
-func (a *Adapter) postMessage(msg wabbit.Delivery) error {
-	a.logger.Info("url ->" + a.httpMessageSender.Target)
-	req, err := a.httpMessageSender.NewCloudEventRequest(a.context)
-	if err != nil {
-		return err
+func msgContentType(msg wabbit.Delivery) (string, error) {
+	var contentType string
+
+	for key, val := range msg.Headers() {
+		if strings.ToLower(key) == "content-type" {
+			contentType = strings.ToLower(fmt.Sprint(val))
```

Comment:
instead of fmt.Sprint, I wonder if we should be doing the type assertion, or at least reading here I'd think that's what we should be doing?
https://github.com/streadway/amqp/blob/master/types.go#L221